core/src/main/java/cz/xtf/core/config/WaitingConfig.java

```diff
@@ -5,9 +5,11 @@
 public class WaitingConfig {
 	public static final String WAITING_TIMEOUT = "xtf.waiting.timeout";
 	public static final String WAITING_LOG_LEVEL = "xtf.waiting.log.level";
+	public static final String CLEAN_PROJECT_TIMEOUT = "xtf.clean.project.timeout";
```

Comment:
@mchoma Can you make it `xtf.waiting.timeout.cleanup` (alternatively `clean`, `clean_project`, `project_cleanup`)? Property will be more consistent with others. Also I think `cleanup` should be enough, I can't recall any other cleanup we do in the framework atm.

Reply:
I have changed to `xtf.waiting.timeout.cleanup` please have a look